src/wallet.ts

```diff
@@ -100,16 +100,17 @@ export class Wallet implements WalletInterface {
     recipient: string,
     amount: number,
     asset: string,
-    changeAddress: string
+    changeAddress: string,
+    satsPerByte: number = 0.1
   ): string {
     const pset = decodePset(psetBase64);
 
-    // at 0.1 sat per byte means pretty big transactions
-    const FIXED_FEE = 2000;
     let inputBlindingKeys: Array<Buffer> = [];
     let outputBlindingKeys: Array<Buffer> = [];
 
-    let lbtcAmountToLookup = FIXED_FEE;
+    const feeEstimation = Math.ceil(estimateTxSize(1, 2) * satsPerByte);
```

Comment:
Will ad a comment to makes clear this is the default bare minimum of a common tx